src/bin/miri.rs

```diff
@@ -19,6 +19,7 @@ use std::env;
 
 use rustc_interface::interface;
 use rustc::hir::def_id::LOCAL_CRATE;
+use hex::FromHexError;
```

Comment:
We usually separate the blocks (std, rustc, other) with an empty line.

Reply:
I changed it to the style from `src/intptrcast.rs` - `std`, `other`, `rustc`.

Reply:
Good catch!